canonical-web-and-design/maas-ui
# refactor(ui): refactor filesystems table to work with controllers

## Done

- Refactored filesystems table to work with controllers

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Check that the filesystems table is unchanged for machine
  - Test out running a few actions on a machine that is in Ready or Allocated state
- Go to the React storage tab of a controller and check that the filesystems table data matches the AngularJS version 

## Fixes

Fixes canonical-web-and-design/app-tribe#996

## Screenshot

![Screenshot 2022-05-31 at 17-23-27 karura storage karura MAAS](https://user-images.githubusercontent.com/25733845/171116017-7a0f1929-0062-4c58-86fb-7f066a33293d.png)
